conf/messages.zh-TW

```diff
@@ -65,7 +65,6 @@ city.name.pittsburgh-pa = 匹茲堡
 city.name.chicago-il = 芝加哥
 city.name.amsterdam = 阿姆斯特丹
 city.name.la-piedad = 拉彼達
-city.name.la-piedad-old = 拉彼達(old)
```

Comment:
Looks like when merging changes from develop, you may have done more of a "keep my version" rather than a true "merge"! There are a couple lines changed here that shouldn't be. IIRC I changed lines only in the zh-TW file...